benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -103,8 +113,9 @@ def _finalize_vm(self):
         metric_results = self._prometheus_metrics_operation.run_prometheus_queries()
         prometheus_result = self._prometheus_metrics_operation.parse_prometheus_metrics(data=metric_results)
         # update total vm run time
-        total_run_time = self._get_bootstorm_vm_total_run_time()
-        self._data_dict.update({'total_run_time': total_run_time})
+        if not self._verification_only:
+            total_run_time = self._get_bootstorm_vm_total_run_time()
+            self._data_dict.update({'total_run_time': total_run_time})
```

Comment:
It wouldn't be a bad idea to stick some distinctive value in there even in verification mode, such as -1, so that the schema is the same both ways.

Reply:
total_run_time will be empty when no input data

Reply:
Understood, but that's about testing that total_run_time is handled correctly.  Not that big of a deal, but think about it.

Reply:
I prefer that it will be empty in ElasticSearch instead of -1, more simple to handle it.